data/building-hours/9-osa.yaml

```diff
@@ -0,0 +1,17 @@
+name: OSA
+subtitle: Office of Student Activities
```

Comment:
I feel that @hawkrives has a decent point. I might flip the name and subtitle. I bet it is thought of as the "SGA office", so an expansion couldn't hurt.